src/core/shutdown.c

```diff
@@ -256,6 +256,19 @@ static void sync_with_progress(void) {
         (void) kill(pid, SIGKILL);
 }
 
+static void disable_coredumps(void) {
+        int fd;
+
+        fd = open("/proc/sys/kernel/core_pattern", O_WRONLY|O_NOCTTY|O_CLOEXEC);
+        if (fd < 0)
+                return;
+
+        write(fd, "/dev/null\n", 10);
+        close(fd);
```

Comment:
please use `(void) write_string_file("/proc/sys/kernel/core_pattern", "|/bin/false", 0);` or so for this.

That said, we have similar code at two other places already. Could you please introduce a helper call in src/basic/util.c which is called "disable_core_dumps()" or so that wraps this and then is used here and for the other two cases.

Reply:
And the new function should do the container check internally, so that we have it implicitly everywhere...